R/foreign-keys.R

```diff
@@ -502,7 +502,7 @@ check_fk <- function(t1, t1_name, colname, t2, t2_name, pk) {
 
   t1_join <-
     t1 %>%
-    count(!!!t1_vals) %>%
+    count(!!!t1_vals, name = "*n*") %>%
```

Comment:
See `unique_prefix()` for an existing alternative approach to names that are guaranteed to be unique.